### DISCUSSION TITLE
AWS Disadvantages

### DISCUSSION THREAD

**Discussion by abhijna** [nih-cfde/training-and-engagement#388] [AWS] [answered]
What are some disadvantages of using AWS?

**Comment by abhijna** [comment, marked as answer]
- Expensive in the long run
- Training and skill: maybe a deterrent to collaborators

**Reply by ctb** [reply]
I've heard estimates that it's between 5x and 10x more expensive than an HPC. I think this relies on the HPC being essentially 100% utilized, however.
